pages/DynamicPOC/index.tsx

```diff
@@ -218,7 +204,7 @@ const DynamicPOC: React.FC<{ questions: Question[] }> = ({ questions }) => {
                   updateFormValues(values)
                 }
                 _handleNext()
-                if (currentQuestion.type === 'RESULT') {
+                if (currentQuestion.type === QuestionType.RESULT) {
                   _handleSubmit(values)
                   setSubmitting(false)
                 }
```

Comment:
Would it be worth pulling this into its own function instead of putting it down by where the more html-y stuff is?

Reply:
imo not worth in this PR, will cause some major merge conflicts in the styling PR too